packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -92,6 +112,10 @@ export class BlockList extends Component {
 		this.scrollViewRef = ref;
 	}
 
+	setListRef( ref ) {
+		this.listRef = ref;
+	}
+
```

Comment:
Is this being used?

Reply:
It was being used before but not anymore. I will remove it 🙏